### PR TITLE
fix(trading): cropping of sidebar elements

### LINKED ISSUE #6484: Slider scale is trimmed on the left and right side
## Description

Slider scale is trimmed on the left and right side

## Steps to Reproduce

1. Go to trading
2. Check slider scale on the deal ticket 
3. See error: Slider scale is trimmed on the left and right side

## Expected behavior
Slider scale is  not trimmed on the left and right side

## Screenshots
image

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #6484  

# Description ℹ️

Fix the slider scale being trimmed on the left and right sides. 

# Demo 📺

image

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

**Comment by daro-maj** [comment]
Can we also fix tooltip ? 
image

**Comment by MadalinaRaicu** [comment]
> Can we also fix tooltip ? image


Yes, but it will be a little moved to the right   


Screenshot 2024-06-06 at 12 43 35

File: libs/ui-toolkit/src/components/slider/percentage-slider.tsx [hunk 1]
```diff
@@ -86,9 +88,11 @@ export const PercentageSlider = ({
                     }
                   : undefined
               }
-              className="absolute translate-x-[-50%] text-xs font-alpha mt-1 text-vega-clight-100 dark:text-vega-cdark-100"
+              className={classNames(
+                'first:left-[1%] last:left-[98%] md:last:left-[99%] lg:last:left-[97%] absolute translate-x-[-50%] text-xs font-alpha mt-1 text-vega-clight-100 dark:text-vega-cdark-100'
```
Comment (mattrussell36) [inline libs/ui-toolkit/src/components/slider/percentage-slider.tsx:92]: I think theres a better way to do this. This looks like a pain to maintain

**Review by mattrussell36 — CHANGES_REQUESTED** [review]

File: libs/ui-toolkit/src/components/slider/percentage-slider.tsx (hunk 1 above)
Comment (mattrussell36) [reply libs/ui-toolkit/src/components/slider/percentage-slider.tsx:92]: The way the scroll area for the deal ticket was set up wasn't super great which resulted in the cropping. We can just edit the padding of the scroll container, and avoid all the complexity of ensuring all the numbers line up with the slider indicators

```
const ContentWrapper = ({
  children,
  title,
}: {
  children: ReactNode;
  title?: string;
}) => {
  return (
    <TinyScroll className="py-4" data-testid="sidebar-content">
      {title && <h2 className="mb-4 px-3">{title}</h2>}
      <div className="px-3">{children}</div>
    </TinyScroll>
  );
};

```

Comment (MadalinaRaicu) [reply libs/ui-toolkit/src/components/slider/percentage-slider.tsx]: updated

**Review by daro-maj — APPROVED** [review]

**Review by mattrussell36 — APPROVED** [review]

**Pull request merged**